_vendor/src/github.com/gorilla/context/doc.go

```diff
@@ -5,6 +5,12 @@
 /*
 Package context stores values shared during a request lifetime.
 
+Note: gorilla/context, having been born well before `context.Context` existed,
```

Comment:
I think we should not use `~1.1.0`, but use a special commit version in glide.yaml.